gym/core.py

```diff
@@ -85,9 +92,9 @@ def reset(
         Returns:
             observation (object): the initial observation.
         """
-        # Initialize the RNG if it's the first reset, or if the seed is manually passed
-        if seed is not None or self.np_random is None:
-            self.np_random, seed = seeding.np_random(seed)
+        # Initialize the RNG if the seed is manually passed
+        if seed is not None:
+            self.seed(seed)
```

Comment:
What's the reason for this? `self.seed` is explicitly deprecated

Reply:
with removal of uninitialized _np_random, this was pure code duplication, so I used self.seed(). Didn't know its deprecated.